tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/MsTestIntegration.cs

```diff
@@ -177,13 +186,13 @@ private static Dictionary<string, List<string>> GetTraits(MethodInfo methodInfo)
                     testProperties ??= new Dictionary<string, List<string>>();
                     if (!testProperties.TryGetValue("Category", out var categoryList))
                     {
-                        categoryList = new List<string>();
+                        categoryList = [];
                         testProperties["Category"] = categoryList;
                     }
 
                     if (tattr.TryDuckCast<TestCategoryAttributeStruct>(out var tattrStruct))
                     {
-                        categoryList.AddRange(tattrStruct.TestCategories);
+                        categoryList.AddRange(tattrStruct.TestCategories ?? []);
```

Comment:
nit: prob better moving an `is null` check to the `if` as `AddRange` still has to do more work if it's passed an empty collection

Reply:
true, done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)